pkg/oci/mutate/mutate.go

```diff
@@ -211,6 +211,13 @@ func (si *signedImage) Attestations() (oci.Signatures, error) {
 			return base, nil
 		}
 	}
+	if si.so.ro != nil {
+		replace, err := si.so.ro.Replace(base, si.att)
```

Comment:
Looks like these paths aren't covered by current unit tests, can we get some unit tests for these, please? 🙏